src/Centreon/Infrastructure/PlatformTopology/PlatformTopologyRegisterRepositoryAPI.php

```diff
@@ -61,6 +68,74 @@ public function __construct(HttpClientInterface $httpClient, ApiPlatform $apiPla
         $this->apiPlatform = $apiPlatform;
     }
 
+    /**
+     * Get a valid token to request the API.
+     *
+     * @param Platform $platform
+     * @param PlatformInformation $platformInformation
+     * @param Proxy $proxyService
+     * @return string
+     */
+    private function getToken(
+        Platform $platform,
+        PlatformInformation $platformInformation,
+        Proxy $proxyService = null
+    ): string {
+        // Central's API endpoints base path building
+        $this->baseApiEndpoint = $platformInformation->getApiScheme() . '://'
+            . $platformInformation->getCentralServerAddress() . ':'
+            . $platformInformation->getApiPort() . '/'
+            . $platformInformation->getApiPath() . '/api/v'
+            . ((string) $this->apiPlatform->getVersion()) . '/';
```

Comment:
IMHO, base endpoint should not contain trailing slash
It's like `__DIR__` constant, it does not add trailing slash

Reply:
I'm not pretty sure if I understood well your feedback, I've changed he trailing slash for a DIRECTORY_SEPARATOR constant.